File: frontend/src/constants/ReleaseConclusion.ts
export const ReleaseConclusion = ["RELEASABLE","BLOCKED"] as const;
export type ReleaseConclusion = (typeof ReleaseConclusion)[number];
export const ReleaseConclusionText: Record<ReleaseConclusion, string> = {
  RELEASABLE: "可放行",
  BLOCKED: "暂不可放行"
};
export const ReleaseBlockerKindText: Record<string, string> = {
  TASK: "未完成任务",
  DELAY: "未关闭延误",
  BOOKING: "预约冲突"
};

File: backend/src/constants/ReleaseConclusion.go
package constants

// ReleaseConclusion 放行结论：汇总页与提交放行共用同一套判定。
var ReleaseConclusion = []string{"RELEASABLE", "BLOCKED"}

const (
	ReleaseConclusionReleasable = "RELEASABLE"
	ReleaseConclusionBlocked    = "BLOCKED"
)

// 放行复核的三类条件标识，用于 blocker 的 kind 字段。
const (
	ReleaseBlockerKindTask    = "TASK"
	ReleaseBlockerKindDelay   = "DELAY"
	ReleaseBlockerKindBooking = "BOOKING"
)
